tools/ovc/unit_tests/moc_tf_fe/conversion_basic_models_test.py

```diff
@@ -169,20 +60,21 @@ def test_cutting_fp32(self, input_freezing_value, inputs, expected,
 
     @generate(
         *[
-            (
-                    "x[1,4],y[4]",
-                    {"x": np.array([[3, 2, 1, 5]], dtype=np.int32), "y": np.array([0, -1, -7, 8], dtype=np.int32)},
-                    np.array([[3, 1, -6, 13]], dtype=np.int32),
-                    np.int32,
-                    None
-            ),
-            (
-                    "x,y",
-                    {"x": np.array([[-3, 20, 1]], dtype=np.int32), "y": np.array([[10, -11, -17]], dtype=np.int32)},
-                    np.array([[7, 9, -16]], dtype=np.int32),
-                    np.int32,
-                    None
-            ),
+            # TODO: Return this test when new 'cut' helper is introduced
+            # (
+            #         "x[1,4],y[4]",
```

Comment:
why do we need cut helper if there is no cutting?

Reply:
Answered above.